Read-and-Save-document/Open-and-save-Word-document/Blazor/Server-side-application/Open-and-save-Word-document/Data/WordService.cs.bak

```diff
@@ -0,0 +1,28 @@
+﻿using Syncfusion.DocIO;
```

Comment:
Remove this unwanted file WordService.cs.bak

Reply:
Addressed the requested changes